### PR TITLE
chore(ci) setup travis-ci tests for arm64 as well as amd64

### PR DESCRIPTION
**Pull request by hutchic**
I did this to try and reproduce a bug -- and it didn't though that's not too surprising. I don't have strong feelings if we merge it or not

### CODE REVIEW

**Comment by kikito** [comment]
Approved, but Travis doesn't like the changes.

**Comment by dndx** [comment]
`1.13.6.*` and before OpenResty doesn't support ARM64, they should be disabled in the matrix but otherwise this PR and test results looks fine.

File: .travis.yml [hunk 1]
```diff
@@ -10,16 +10,45 @@ compiler: gcc
...
 
+arch:
+  - amd64
+  - arm64
+matrix:
+  exclude:
+   - arch: arm64
+     env: OPENRESTY=1.13.6.2
+   - arch: arm64
+     env: OPENRESTY=1.13.6.2
```
Comment (javierguerragiraldez) [inline .travis.yml]: duplicate?

Comment (hutchic) [reply .travis.yml]: yup good catch

**Comment by thibaultcha** [comment]
Thanks for spending time investing the issue. Given this is a pure Lua library (no C, no FFI), there is no point in maintaining an ARM64 test mode. If any issue would arise from ARM64, they'd be coming from a dependency of this library (e.g. ngx_lua, lua-resty-lrucache, LuaJIT...).

**Pull request closed without merging**